internal/tiltfile/live_update_test.go

```diff
@@ -229,19 +230,20 @@ func (f *liveUpdateFixture) init() {
 	f.dockerfile("foo/Dockerfile")
 	f.yaml("foo.yaml", deployment("foo", image(f.configuredImageName)))
 
-	tiltfile := fmt.Sprintf(`
-%s
-k8s_resource('foo', 'foo.yaml')
-live_update('%s',
-  [
+	luSteps := `[
+    fall_back_on(['foo/i', 'foo/j']),
 	sync('foo/b', '/c'),
 	run('f', ['g', 'h']),
 	restart_container(),
-  ],
-  ['foo/i', 'foo/j']
-)`, f.tiltfileCode, f.configuredImageName)
+]`
+	codeToInsert := fmt.Sprintf(f.tiltfileCode, luSteps)
+	tiltfile := fmt.Sprintf(`
+k8s_yaml('foo.yaml')
+%s
+`, codeToInsert)
 	f.file("Tiltfile", tiltfile)
 
+	fmt.Println(tiltfile)
```

Comment:
if this is intentional rather than temporary debug, it might be better to `fmt.Printf("tiltfile:\n%q\n", tiltfile)`

Reply:
BOLD ASSUMPTION.